conductr_cli/bundle_scale.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import unicode_literals
+from conductr_cli import conduct_url, sse_client
+from conductr_cli.exceptions import WaitTimeoutError
+from datetime import datetime
+import json
+import logging
+import requests
+
+
+def get_scale(bundle_id, args):
+    bundles_url = conduct_url.url('bundles', args)
+    response = requests.get(bundles_url)
+    response.raise_for_status()
+    bundles = json.loads(response.text)
+    matching_bundles = [bundle for bundle in bundles if bundle['bundleId'] == bundle_id]
+    if matching_bundles:
+        matching_bundle = matching_bundles[0]
+        if 'bundleExecutions' in matching_bundle:
+            return len(matching_bundle['bundleExecutions'])
+
+    return 0
+
+
+def wait_for_scale(bundle_id, expected_scale, args):
+    log = logging.getLogger(__name__)
+    start_time = datetime.now()
+
+    bundle_scale = get_scale(bundle_id, args)
+    if bundle_scale == expected_scale:
+        log.info('Bundle {} expected scale {} is met'.format(bundle_id, expected_scale))
+        return
+    else:
+        log.info('Bundle {} waiting to reach expected scale {}'.format(bundle_id, expected_scale))
+        bundle_events_url = conduct_url.url('bundles/events', args)
+        sse_events = sse_client.get_events(bundle_events_url)
```

Comment:
Don't you want to call get_scale upon having successfully connected so that you can be certain to not have missed any events?